src/style.css

```diff
@@ -1 +1,51 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__header {
+  display: flex;
+  padding: 0 40px;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 10px;
+}
+
+.nav__items {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin: 0 10px;
+  padding: 23px 0 19px;
+  color: #000;
+  border-bottom: 4px solid #fff;
+}
+
+.nav__item:hover {
+  color: #00acdc;
+}
+
+.nav__item:active {
+  border-bottom: 4px solid #00acdc;
+  color: #00acdc;
+}
```

Comment:
Use ::after for adding bottom line for active link (don't forget to add border-radius - it should fix your tests)